Makefile

```diff
@@ -148,12 +149,15 @@ build: install build-$(CALYPSO_ENV)
 
 build-css: public/style.css public/style-rtl.css public/style-debug.css public/editor.css
 
-build-development: build-server $(CLIENT_CONFIG_FILE) server/devdocs/search-index.js build-css
+build-inline-js:
+	@$(UGLIFYJS) -o public/catch-js-errors.js catch-js-errors.js
```

Comment:
Do we need a separate build process for this one file? Can we fit into the current build process?